ports/javascriptcore/vcpkg.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "javascriptcore",
+  "version": "6.0",
+  "port-version": 10,
+  "description": "JavaScriptCore is the built-in JavaScript engine for WebKit, which implements ECMAScript as in ECMA-262 specification.",
+  "supports": "!uwp & !static & windows & x64",
+  "dependencies": [
```

Please add license and homepage.

---

@Adela0814 Adding homepage but according to https://learn.microsoft.com/en-us/vcpkg/reference/vcpkg-json#license, will have to leave license as empty.
As the license is a combination of LICENSE-APPLE and LICENSE-LGPL-2.1 
And that combination is provided as `copyright` file.

---

This is not resolved. The proper change is:
```suggestion
  "license": null,
  "dependencies": [
```
where `null` means: Too complex, please read the copyright file.